### PR TITLE
add a new console.log()

### PR DESCRIPTION
**Pull request by rohan-sorkar**
in this description, I should say-- what, why, and how everything happens

### CODE REVIEW

File: hello.js [hunk 1]
```diff
@@ -1 +1,2 @@
 console.log('hello world');
+console.log('hello world is not a hello world');
```
Comment (mohammadsaju) [inline hello.js:2]: could you change 'hello world is not a hello world' to 'hello world is a hello world'

**Review by mohammadsaju — CHANGES_REQUESTED** [review]

File: hello.js (hunk 1 above)
Comment (rohan-sorkar) [reply hello.js:2]: yes i can

Comment (mohammadsaju) [inline hello.js:2]: could you please add another console.log('you are a programmer')

**Review by mohammadsaju — COMMENTED** [review]
could you please add another line of console.log('you are a programmer')

**Comment by rohan-sorkar** [comment]
yeah sure

File: hello.js (hunk 1 above)
Comment (mohammadsaju) [inline hello.js:2]: can you replace it

File: hello.js [hunk 2]
```diff
@@ -1 +1,2 @@
 console.log('hello world');
```
Comment (mohammadsaju) [inline hello.js:1]: replace the string with 'hello d'

File: hello.js (hunk 1 above)
Comment (rohan-sorkar) [reply hello.js:2]: sure

File: hello.js (hunk 2 above)
Comment (rohan-sorkar) [reply hello.js:1]: okk

**Review by mohammadsaju — CHANGES_REQUESTED** [review]
please change the code

**Review by mohammadsaju — APPROVED** [review]
well done @rohan-sorkar

**Pull request merged**